components/definition/populations/NetPyNEPopulation.js

```diff
@@ -124,51 +137,48 @@ export default class NetPyNEPopulation extends React.Component {
           <NetPyNEField id="netParams.popParams.cellType" >
             <PythonControlledTextField
               model={"netParams.popParams['" + this.props.name + "']['cellType']"}
-              id={"popCellType"}
             />
           </NetPyNEField>
-
+          
           <NetPyNEField id="netParams.popParams.cellModel" >
-            <PythonControlledAutoComplete
-              dataSource={[]}
+            <PythonControlledTextField
```

Comment:
Did you do this on purpose? If so, can you expand the PR description with the changes you are doing apart from the casper extension?

Reply:
yes, this is an important one. The auto-complete in cellModel behaves very weird. You enter a value, and the field goes empty, then the value reappears, then it goes empty again, then it triggers a refresh again, and finally it gets empty. 
If you try to close the card and re-open it, you get the same behavior. It was imposible to assert the value of the fields in the casper-test. I had to change it.

Reply:
Ok, we can also try to fix instead of just replacing it. So... aren't we using autocomplete anywhere else in the code?

Reply:
no, that is the only field that uses autocomplete. I will create a card for that so we don't forget